includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -1167,6 +1169,25 @@ public static function force_allow_subscription_switching( $can_switch, $option_
 		return 'yes';
 	}
 
+	/**
+	 * Force option for allowing retries for failed payments to ON unless the
+	 * NEWSPACK_PREVENT_WC_ALLOW_FAILED_PAYMENT_RETRIES_OVERRIDE constant is set.
+	 * 
+	 * See: https://woo.com/document/subscriptions/failed-payment-retry/
+	 * 
+	 * @param bool   $should_retry Whether WooCommerce should automatically retry failed payments.
+	 * @param string $option_name The name of the option.
+	 *
+	 * @return string Option value.
+	 */
+	public static function force_allow_failed_payment_retry( $should_retry, $option_name ) {
+		if ( defined( 'NEWSPACK_PREVENT_WC_ALLOW_FAILED_PAYMENT_RETRIES_OVERRIDE' ) && NEWSPACK_PREVENT_WC_ALLOW_FAILED_PAYMENT_RETRIES_OVERRIDE ) {
+			return $can_switch;
```

Comment:
```suggestion
			return $should_retry;
```

Reply:
nit. if we don't need `option_name` here, we can remove it from the signature and the `2` from `add_filter`

Reply:
Yes, I removed the unused arg in 4d99f80 but forgot to remove it from the action callback definition. Fixed that in 36b10f1a